app/operations/assignments/create.rb

```diff
@@ -22,7 +22,7 @@ def execute
         subjects: []
       }
 
-      panoptes.add_subjects_to_subject_set(subject_set["id"], subject_ids)
+      FillSubjectSetWorker.perform_async(subject_set["id"], subject_ids)
```

Comment:
Query: where is `perform_async` defined? Is this implicit in the Ruby naming - i.e. by adding `_async` it makes the `.perform()` function asynchronous?